openedx/core/djangoapps/user_api/accounts/signals.py

```diff
@@ -4,4 +4,14 @@
 
 from django.dispatch import Signal
 
+# Signal to retire a user from third party mailing services, such as Sailthru.
+USER_RETIRE_THIRD_PARTY_MAILINGS = Signal(providing_args=["user"])
+
+# Signal to retire a user from LMS-initiated mailings (course mailings, etc)
 USER_RETIRE_MAILINGS = Signal(providing_args=["user"])
+
+# Signal to retire LMS critical information
+USER_RETIRE_LMS_CRITICAL = Signal(providing_args=["user"])
```

Comment:
I'd argue that since that call is the one that changes the username and marks them inactive it's pretty critical. Or at least as critical as anything in retirement. I was hoping it would encourage people to add any new functionality to the MISC handler and keep that one as simple as possible. I'm open to changing it, though, just want to make it clear that this is the about the most major step after deactivate/logout.

Reply:
ok - fine with me